invenio_rdm_records/resources/serializers/ui/schema.py

```diff
@@ -217,6 +217,25 @@ class MeetingSchema(Schema):
     url = SanitizedUnicode()
 
 
+def add_community_information(obj, _):
```

Comment:
this information is already available in rest API, why do we need to add it?

Reply:
Ah it was because of this https://codimd.web.cern.ch/iwBGs2YXQruUt_H2ZmFXZA?view#Errors-display